src/app/Models/textarea/textarea.component.html

```diff
@@ -1,9 +1,24 @@
 <div [class]="wrapperClassName">
   <label *ngIf="label" class="form__label"> {{ label }} </label>
-  <textarea
-    class="form__textarea"
-    [class]="className"
-    placeholder="{{ placeholder?.length ? placeholder : placeholder }}"
-    [formControl]="control"
-  ></textarea>
+  <ng-container *ngIf="!isEditable; else editableTextArea">
+    <textarea
+      class="form__textarea"
+      [class]="className"
+      placeholder="{{ placeholder?.length ? placeholder : placeholder }}"
```

Comment:
create a method for in the ts file to get the placeholder and use property binding instead of interpolation.